src/FunderMaps.Webservice/Startup.cs

```diff
@@ -86,9 +94,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             }
             else
             {
-                //app.UseExceptionHandler("/oops");
+                app.UseExceptionHandler(); // TODO This should be fixed for production!
```

Comment:
What does that mean?

Reply:
This means that we want to re-consider the exception handling pipeline since our custom exception handler catches no exception, and the default exception mapper exposes the stacktrace.